inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/heartbeat/HeartbeatManager.java

```diff
@@ -79,7 +84,7 @@ public void start() {
     }
 
     @Override
-    public void reportHeartbeat(HeartbeatMsg heartbeat) {
+    public void reportHeartbeat(List<HeartbeatMsg> heartbeat) {
```

Comment:
Why report multiple heartbeats?

A DataProxy node can start multiple different sources but does not need to construct a heartbeat message for each source

Reply:
Has been removed.